src/tokenOperations/tokenOperationsFactory.ts

```diff
@@ -12,6 +12,7 @@ interface IConfig {
     minGasLimit: IGasLimit;
     gasLimitPerByte: IGasLimit;
     gasLimitIssue: IGasLimit;
+    gasLimitToggleBurnRoleGlobally: IGasLimit;
```

Comment:
In theory, this is a breaking change. However, in practice, it should not be. Generally speaking, users would be using the default config implementation, `TokenOperationsFactoryConfig`. Furthermore, this transactions factory isn't widely used yet.